Makefile.toml

```diff
@@ -0,0 +1,35 @@
+extend=[
+    { path = "grcov.makefile.toml" }
+]
+
+[tasks.format]
+install_crate = "rustfmt"
+command = "cargo"
+args = ["fmt", "--all", "--", "--check"]
+
+[tasks.clippy]
+command = "cargo"
+args = ["clippy", "--tests", "--", "-D","warnings", "-D", "clippy::pedantic"]
+
+[tasks.coverage]
+alias="coverage_grcov"
+
+[tasks.doctest]
+command = "cargo"
+args = ["test", "--doc"]
```

Comment:
cargo test runs doc tests already, so aren't you doubling up on that work?

Reply:
you're right, `cargo test` does run doc tests, so this is not required